SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/CrossServerRPCService.cpp

```diff
@@ -152,7 +169,10 @@ void CrossServerRPCService::EntityAdded(const Worker_EntityId EntityId)
 		OnEndpointAuthorityGained(EntityId, Component);
 	}
 	CrossServerEndpoints* Endpoints = CrossServerDataStore.Find(EntityId);
-	HandleRPC(EntityId, Endpoints->ReceivedRPCs.GetValue());
+	if (Endpoints->ReceivedRPCs.IsSet())
```

Comment:
Not sure I follow this change?

Reply:
Could do. Added hidden feature of the worker entity being able to receive RPC might even be useful.